### DISCUSSION TITLE
Add Close and Delete to the toolbar

### DISCUSSION THREAD

**Discussion by giwul70** [sumatrapdfreader/sumatrapdf#3548] [Ideas]
Suggestion:  add a Close+Delete (or just a [Delete]) button to the toolbar. 
To avoid accidental delete, the button to be placed away from others and not in the menu.
Right now, deleting a file means: close, launch Explorer, click, click browse, click click to the right folder, scroll to the right file and select delete from there. That is the way I do it. 
The button makes it more convenient to [Delete]  the file.

Just a suggestion.
Thanks!

![SumatraPDF-Reader-suggestion-close and delete button-23062023 150126](https://github.com/sumatrapdfreader/sumatrapdf/assets/101186455/632431a7-76f8-4788-abe3-b2338ec8566c)

**Comment by GitHubRulesOK** [comment]
for large PDF (over 32 MB) it is not generally possible to delete since Windows locks any file as in use
The exception for essential use by PDFLaTeX  is that SumatraPDF and other synchronised Readers (NOTABLY NOT ACROBAT) will un-lock the file alowing accidental or necessary intentional deletion while LaTeX re-writing (in reality its not deleted simply like all del commands its remained as not there)

The simplest approach is save current filename to a list and after SumatraPDF has closed all its settings etc THEN deleted the undesirables.
Examples have been given in the past how that's done using a custom 3key salute.

The secondary benefit is you can review the list before hitting/run delete all at high speed.
